internal/server/server.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+// Version information, assigned by ldflags
+var (
+	CommitHash string
+	BuildDate  string
+)
+
+// Server is the wrapper of http.Server
```

Comment:
It won't be in it's final form :) I'd just say - "Server contains all the necessary information to run Bifrost"